pkg/utils/common.go

```diff
@@ -27,3 +28,17 @@ func WaitUntil(out io.Writer, poll, timeout time.Duration, checkDone func() erro
 	}
 	return fmt.Errorf("timeout reached %s", timeout.String())
 }
+
+type callback func()
+
+func CaptureStdout(c callback) string {
+	r, w, _ := os.Pipe()
+	tmp := os.Stdout
+	os.Stdout = w
+	c()
+	w.Close()
+	stdout, _ := ioutil.ReadAll(r)
+	os.Stdout = tmp
```

Comment:
Seems like resetting `os.Stdout` to its original value should be deferred in case the callback errors out.